### PR TITLE
New package: ntfy-2.0.0

### PR DESCRIPTION
**Pull request by TinfoilSubmarine**
#### Testing the changes
- I tested the changes in this PR: **YES**

Closes https://github.com/void-linux/void-packages/issues/41422

I am using this exclusively for the server functionality as I am using the ntfy android app as the client. @biopsin did you require any additional services for the client side?

### CODE REVIEW

**Comment by biopsin** [comment]
@TinfoilSubmarine 
Whosh that was fast, thougth I would tinker with this upcomming weekend.
Honestly I just discovered it and have not had the time to investigate all aspects, so Im unaware off any services I might require for the moment. Tnx.

**Comment by TinfoilSubmarine** [comment]
Don't give me too much credit for it, I had just been working on it the week before :)

For now it just has the service for running the server (meaning `ntfy serve`) since that was the functionality I needed--it seems that the Gotify-UP android app no longer exists and so I need a new UnifiedPush solution.

**Comment by TinfoilSubmarine** [comment]
OK, I fixed the webapp not showing. As far as a client service replicating the [systemd service in the upstream source](https://github.com/binwiederhier/ntfy/blob/main/client/ntfy-client.service), I don't know if this makes much sense since likely a user would just want to have a per-user runit service.

**Comment by TinfoilSubmarine** [comment]
Added a data directory so the server can use the cache/auth file.

**Comment by Piraty** [comment]
is this ready? (if not, please prefix `WIP` to the title until it is)

**Comment by TinfoilSubmarine** [comment]
> is this ready? (if not, please prefix `WIP` to the title until it is)

Yes, it is ready

File: srcpkgs/ntfy/files/ntfy-server/log/run [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/bin/vlogger
```
Comment (Piraty) [inline srcpkgs/ntfy/files/ntfy-server/log/run:1]: see https://github.com/void-linux/void-packages/pull/42026

**Review by Piraty — CHANGES_REQUESTED** [review]

File: srcpkgs/ntfy/files/ntfy-server/log/run (hunk 1 above)
Comment (TinfoilSubmarine) [reply srcpkgs/ntfy/files/ntfy-server/log/run]: Fixed, thanks

**Pull request merged**